# service type for image geometry projection back to 3d world
float64 x
float64 y
---
float64 x
float64 y
float64 z
